MANIFEST.in

```diff
@@ -1,24 +1,31 @@
 prune asset
-include *.md
-include *.py
-include *.txt
-include *.yaml
-include *.yml
+prune zuul.d
+prune .github
+exclude .git_archival.txt
+exclude playbooks
+exclude snap
+include src/molecule/test/functional/.ansible-lint
```

Comment:
It's pointless to enumerate each file since they are already autoincluded by setuptools-scm.